src/connectors/connector_compute.workflows.json

```diff
@@ -12,7 +12,7 @@
               "zone": "us-central1-a"
             },
             {
-              "instance": "[fill in an instance name]"
+              "instance": "INSTANCE_NAME"
```

Comment:
here too, maybe an example?

Reply:
Ack. Unfortunately, we can't have comments in the JSON files.